course_catalog/etl/youtube.py

```diff
@@ -82,6 +84,67 @@ def get_videos_for_playlist(youtube_client, playlist_id, offered_by):
             break
 
 
+def get_captions_for_video(video):
+    """
+    Function which fetches and returns xml captions for a video object
+
+    Args:
+        video (object) : Video Object
```

Comment:
I think this is actually a `course_catalog.models.Video` type instead of `object`?